polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -1145,14 +1145,6 @@ MLIRScanner::EmitSYCLOps(const clang::Expr *Expr,
       if (const auto *RD = dyn_cast<clang::CXXRecordDecl>(Func->getParent()))
         if (!RD->getName().empty())
           OptFuncType = RD->getName();
-      if (!OptFuncType) {
```

Comment:
I assume that non-member functions are still either handled as SYCL special op or SYCL call. What would be an example of such a function here, group functions?

Reply:
`sycl::abs` (this warning is displayed when running the `functions.cpp` lit test.